### ISSUE TITLE
Feature request. Sass based theme

### ISSUE THREAD

**Issue by eugene-p** [bcgov/mygovbc-bootstrap-theme#1]
As from version, 4 Bootstraps moves from LESS to SASS it would be great to have a theme based on SASS.

Current SASS version of bootstrap 3 is located @ https://github.com/twbs/bootstrap-sass.

**Comment by GregTurner** [comment]
I agree @eugene-p, LESS was the primary for v3.x but they've switched to SASS for v4.x which basically means LESS's days are numbered.  Bootstrap v4 is in beta which makes it a good time to consider upgrading this repo!

**Comment by GregTurner** [comment]
https://github.com/bcgov/Gov-2.0-Bootstrap-Skeleton/issues/13
